precompiles/ArbAddressTable_test.go

```diff
@@ -47,6 +47,12 @@ func TestAddressTable1(t *testing.T) {
 
 	addr := common.BytesToAddress(crypto.Keccak256([]byte{})[:20])
 
+	exists, err := atab.AddressExists(context, evm, addr)
```

Comment:
This is good, but we also want to make sure each precompile function has a system_test that calls it as part of the EVM and makes sure it doesn't revert.

In arbos31 we had an issue which was a difference between the type minInitGas returned in go vs. the type it retuned in solidity. A unit-test like this would not have caught it.

For functions where we have a reasonable unit test - the system test doesn't need to check functionality too much, just make sure it doesn't revert.

Reply:
Done.